openshift/rh-che.app.yaml

```diff
@@ -85,6 +85,16 @@ objects:
               configMapKeyRef:
                 key: che-jsonrpc-max-pool-size
                 name: rhche
+          - name: CHE_CORS_ENABLED
+            valueFrom:
+              configMapKeyRef:
+                key: che-cors-enabled
+                name: rhche
+          - name: CHE_WSAGENT_CORS_ALLOWED__ORIGINS
```

Comment:
(Totally optional) suggestion: should we move this env entry down to keep the env var names alphabetically ordered? It doesn't matter too much for just one yaml but I've found it useful when comparing against the minishift addon template.

Reply:
no IMO, since those env var would be removed in the `6.16.0` release - CORS is planned to be always disabled

Reply:
ah, sorry last comment was for `Should we also add these values to the minishift template yaml? (I don't know if it's interesting/useful to do testing there)`. Sure thing I will move the env var down ;-)

Reply:
@amisevsk should be better now